src/php/binaries.rs

```diff
@@ -77,23 +77,35 @@ pub(crate) fn all() -> HashMap<PhpVersion, PhpBinary> {
 fn get_all() -> HashMap<PhpVersion, PhpBinary> {
     let mut binaries: HashMap<PhpVersion, PhpBinary> = HashMap::new();
 
-    binaries_from_env(&mut binaries);
     binaries_from_rymfony_env(&mut binaries);
 
-    merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/bin")));
-    merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/sbin")));
+    if cfg!(not(target_os = "macos")) {
+        binaries_from_env(&mut binaries);
+        merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/bin")));
+        merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/sbin")));
+    }
```

Comment:
I think this might conflict with pre-Big Sur versions of MacOS. Maybe we should find a way to detect MacOS's version instead 🤔

Reply:
MacOS users use default php binary ? 
Maybe add a special mention in the readme ?

Reply:
I'm almost certain that some day someone will come and say that they can't use the native binary on some older version of MacOS 🤣

Reply:
I totally agree with you 😄 
I just added a version comparison based on BigSur. What do you think ?

Reply:
Thanks, I left a comment about it 👍